docs/docs/setting-up/data-ingestion/s3.md

```diff
@@ -37,8 +36,8 @@ Let's look closely at the command above:
 
 When a job is submitted, Bacalhau prints out the related `job_id`. We store that in an environment variable so that we can reuse it later on.
 
-:::tip
-This only works with datasets that are publicly accessible and don't require an AWS account or pay to use buckets.
+:::info
+This works with datasets that are either publicly accessible or when node has valid s3 credentials to access such data. See the [S3 Source Specification](../other-specifications/sources/s3.md#credential-requirements) for more details.
```

Comment:
> This works with datasets that are either publicly accessible or when node has valid s3 credentials to access such data

The either in this sentence implies that another attribute of datasets is about to be mentioned.  I think it might be in the wrong place


```
This works either with datasets that are publicly accessible, or with nodes that have valid s3 credentials for the data

```